.circleci/config.yml

```diff
@@ -718,6 +718,39 @@ jobs:
           command: yarn test:ci
       - magma_slack_notify
 
+  nms-e2e-test:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      - NMS_ROOT=${MAGMA_ROOT}/nms/app/packages/magmalte
+    steps:
+      - checkout
+      - docker/install-dc
+      - run: sudo apt-get update -y
+      - run: sudo apt-get install -y apt-transport-https
+      - run: curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - run: echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - run: sudo apt-get update -y
```

Comment:
why run this twice?

Reply:
this is in https://classic.yarnpkg.com/en/docs/install/#debian-stable. 
the first update is to ensure we can install apt-transport-https. It might be a little redundant. I will get rid of it.